app/services/data/get-staging-workload.js

```diff
@@ -11,6 +11,7 @@ const Promise = require('bluebird').Promise
 module.exports = function (range) {
   var omWorkloads = []
   return knex('wmt_extract').whereBetween('wmt_extract.id', range)
+    .leftJoin('t2a', 'wmt_extract.om_key', 't2a.om_key')
```

Comment:
`om_key` does not uniquely define a workload owner. Think you also need to join on team here

Reply:
Examples of this should be in master now.